tests/boot/snapper_rollback.pm

```diff
@@ -18,12 +18,17 @@ use warnings;
 use migration 'check_rollback_system';
 use power_action_utils 'power_action';
 use Utils::Backends 'is_pvm';
+use version_utils;
 
 sub run {
     my ($self) = @_;
-
-    if (!check_screen 'linux-login', 200) {
-        assert_screen 'displaymanager', 90;
+    if (is_leap_migration && check_var('DESKTOP', 'gnome')) {
```

Comment:
Small comments, why leap skip displaymanager ?

Reply:
cause in leap it will boot directly to the desktop as you can see from
https://openqa.nue.suse.com/tests/4743563#step/snapper_rollback/1